AuroraAssetEditor/MainWindow.xaml

```diff
@@ -20,27 +20,27 @@
                   VerticalAlignment="Top"
                   Background="White">
                 <MenuItem ContextMenuOpening="FileOpening" Header="File">
-                    <MenuItem Click="CreateNewOnClick" Header="Create New Asset File" />
-                    <MenuItem Click="LoadAssetOnClick" Header="Load Asset File" />
+                    <MenuItem Click="CreateNewOnClick" Header="Create New Asset Set" />
+                    <MenuItem Click="LoadAssetOnClick" Header="Load Asset(s)" />
                     <MenuItem Click="SaveAllAssetsOnClick" Header="Save All Assets" />
                     <MenuItem x:Name="FtpUpload"
                               Click="SaveAllAssetsFtpOnClick"
                               Header="Save All Assets to FTP" />
                     <Separator />
-                    <MenuItem Click="SaveBoxartOnClick" Header="Save Boxart/Cover Asset" />
-                    <MenuItem Click="SaveBackgroundOnClick" Header="Save Background Asset" />
-                    <MenuItem Click="SaveScreenshotsOnClick" Header="Save Screenshots Asset" />
-                    <MenuItem Click="SaveIconBannerOnClick" Header="Save Icon/Banner Asset" />
+                    <MenuItem Click="SaveBoxartOnClick" Header="Save Cover Art" />
+                    <MenuItem Click="SaveBackgroundOnClick" Header="Save Background" />
+                    <MenuItem Click="SaveIconBannerOnClick" Header="Save Icon &amp; Banner" />
+                    <MenuItem Click="SaveScreenshotsOnClick" Header="Save Screenshots" />
                     <Separator />
                     <MenuItem Click="ExitOnClick" Header="Exit" />
                 </MenuItem>
                 <MenuItem x:Name="SettingsMenu" Header="Settings">
                     <MenuItem x:Name="AutoResizeImages"
-                              Header="Auto Resize Images"
+                              Header="Automatically Resize Images"
                               IsCheckable="True"
-                              IsChecked="True" />
+                              IsChecked="False" />
```

Comment:
Why was this changed to false by default? the default behavior should be to automatically resize images to fit imho.

Reply:
I changed this because I believe it will discourage simply stretching and warping the images. I should have specified this earlier, but I believe that if the end-user sees the warping they'll be encouraged to properly scale their images.

Reply:
Most users will not make their own covers, they'll be finding them online and putting in, therefor to cater to the majority of the user base, i would rather have this returned to the original default

Once you've updated with the adjustments mentioned, your pull request will be accepted/merged

Reply:
Noted, I will push my commit tonight.